blog/2025-02-11-announcing-buildernet-v1.2.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Announcing BuilderNet v1.2
+description: We are excited to announce the release of BuilderNet v1.2! This release introduces fully automatic node bootstrapping, improved system management capabilities and authentication, and lays the groundwork for a more flexible, container-based architecture.
+image: /img/buildernet-cover-photo-m.jpg
+hide_table_of_contents: false
+---
+
+We are excited to announce the release of BuilderNet v1.2, a significant BuilderNet upgrade that includes performance improvements, streamlined operator onboarding, and enhanced security measures.
```

Comment:
"We are excited to release BuilderNet v1.2, a significant upgrade that streamlines operator onboarding and enhances the security and performance of the BuilderNet infrastructure."